tests/json/schema-1-tamps/input.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import psi4
+import json
+
+#Generate JSON data
+
+method = "CCSD"
+basis = "sto-3g"
+json_data = {
+    "schema_name": "qcschema_input",
+    "schema_version": 1,
+    "return_output": True,
+    "molecule": {
+        "geometry": [
+            0.0,
+            0.0,
+            -0.129476941212,
+            0.0,
+            -1.494187339480,
+            1.027446507906,
+            0.0,
+            1.494187339480,
+            1.027446507906
+        ],
+        "symbols": [
+          "O",
+          "H",
+          "H"
+        ],
+        "connectivity" : [
+            (0, 1, 1.0),
+            (0, 2, 1.0)
+        ],
+        "fix_symmetry": "c1"
+    },
+    "driver": "energy",
+    "model": {
+        "method": method,
+        "basis": basis
+    },
+    "keywords": {
+        "scf_type": "df",
+        "mp2_type": "df"
+    },
+    "extras": {
+        "psi4:save_tamps": True
+    }
+}
+
+h2o_test = psi4.geometry("""
+0 1
+units bohr
+O 0.0 0.0 -0.129476941212
+H 0.0 -1.494187339480 1.027446507906
+H 0.0 1.494187339480 1.027446507906
+symmetry c1
+""")
+
+h2o_schema = psi4.core.Molecule.from_schema(json_data)
+
+psi4.compare_values(h2o_test.geometry().to_array(),
+                    h2o_schema.geometry().to_array())
+
+json_ret = psi4.json_wrapper.run_json_qcschema(json_data, True, True)
```

Comment:
```suggestion
json_ret = psi4.schema_wrapper.run_qcschema(json_data, True, True)
```
The `run_json_qcschema` fn you were calling is an internal one called by `run_json` (early effort) and `run_qcschema` (current runner), so it shouldn't be called directly. I bet you can remove the two provenance lines you added after making this change. after this, lgtm!